### ISSUE TITLE
Consider a minor docs re-do when it comes to <Names>

### ISSUE THREAD

**Issue by shahzeb1** [casesandberg/react-color#439]
I couldn't help but notice that the docs say that `You can import AlphaPicker BlockPicker...` but then later in the docs all these elements are referred to as `<Alpha/>`, `<Block/>` and so on. Shouldn't they instead be `<AlphaPicker/>`, `<BlockPicker/>` etc?

**Comment by casesandberg** [comment]
Yes, good point. I am currently working on a new major version 3.0.0, I will track this in that to make sure it gets updated.
